orchagent/main.cpp

```diff
@@ -286,6 +406,35 @@ int main(int argc, char **argv)
         attrs.push_back(attr);
     }
 
+    //Get info required for VOQ system
+    vector<sai_system_port_config_t> sysportconfiglist;
+    if(getSystemPortConfigList(sysportconfiglist))
```

Comment:
I saw

    /* Must be last Attribute */
    attr.id = SAI_REDIS_SWITCH_ATTR_CONTEXT;
    attr.value.u64 = gSwitchId;
    attrs.push_back(attr);

Wonder if we should call this before setting SAI_REDIS_SWITCH_ATTR_CONTEXT?

Reply:
Yes.  Already fixed in previous review comments fix commit 2ca2a00